sihl-user/src/sihl_user.ml

```diff
@@ -5,35 +5,35 @@ let log_src = Logs.Src.create ("sihl.service." ^ Sihl.Contract.User.name)
 module Logs = (val Logs.src_log log_src : Logs.LOG)
 
 module Make (Repo : User_repo.Sig) : Sihl.Contract.User.Sig = struct
-  let find_opt user_id = Repo.get user_id
+  let find_opt = Repo.get
 
-  let find user_id =
-    let%lwt m_user = find_opt user_id in
+  let find ?ctx user_id =
+    let%lwt m_user = find_opt ?ctx user_id in
     match m_user with
     | Some user -> Lwt.return user
     | None ->
       Logs.err (fun m -> m "User not found with id %s" user_id);
       raise (Sihl.Contract.User.Exception "User not found")
   ;;
 
-  let find_by_email_opt email = Repo.get_by_email email
+  let find_by_email_opt ?ctx email = Repo.get_by_email ?ctx email
```

Comment:
Should be 
```ocaml
let find_by_email_opt = Repo.get_by_email
```
for consistency. (Noticed by @mabiede)

Reply:
Thanks @mabiede, fixed.